### DISCUSSION TITLE
Memory usage?

### DISCUSSION THREAD

**Discussion by tikumo** [tomatophp/tomato#23] [Q&A]
I'm trying to install TomatoPHP on my Ubuntu server with PHP 8.2, but I cannot run it with significantly upping the memory. 
Is this normal? Normal Laravel runs fine on 128 MB. Even on 228M it won't start.

Any tips?

**Comment by tikumo** [comment]
![image](https://github.com/tomatophp/tomato/assets/559615/75d8c08f-95c4-430f-a16c-e3077dc3012c)

**Comment by tikumo** [comment]
Was wondering if it was only for the first run, so I upped my server to 500 MB memory but it still runs out.

"Allowed memory size of 448790528 bytes exhausted (tried to allocate 72895842 bytes)"

**Comment by tikumo** [comment]
Folder rights were the problem.. Normal Laravel gave errors but Tomato won't for some reason..

**Comment by fadymondy** [comment]
can you please provide me with your OS and your PHP version and composer version
